VaticanBAV/et/BAVet116.xml

```diff
@@ -67,132 +71,150 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                          </msItem>
-                          <msItem xml:id="ms_i1.10">
-                            <locus from="88v" to="101r"/>
-                            <title type="complete" ref="LIT1100Anapho"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.11">
-                            <locus from="101r" to="112v"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraJohn"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.12">
-                            <locus from="112v" to="123r"/>
-                            <title type="complete" ref="LIT1103Anapho"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.13">
-                            <locus from="123v" to="137v"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraAthanasius"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.14">
-                            <locus from="137v" to="145v"/>
-                            <title type="complete" ref="LIT2040Naakkw"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.15">
-                            <locus from="145v" to="153r"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraEpiphanius"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.16">
-                            <locus from="153r" to="160v"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraJohnChrysostom"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.17">
-                            <locus from="160v" to="164r"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraDioscorus"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.18">
-                            <locus from="164r" to="171v"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraJacobSerug"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.19">
-                            <locus from="171v" to="181r"/>
-                            <title type="complete" ref="LIT1098Anapho"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.20">
-                            <locus from="182v" to="186v"/>
-                            <title type="complete" ref="LIT2038Naakkw"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.21">
-                            <locus from="186v" to="194r"/>
-                            <title type="complete" ref="LIT1569habeka"/>
+
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.2">
+                          <locus from="20v" to="24r"/>
+
```

Comment:
no title here?

Reply:
The two items (20v-22v and 22v-24r) are grouped together in the catalogue as one item, though they are treated separately.

![IMG_0080](https://user-images.githubusercontent.com/6315671/92496585-9b6cbe80-f1f8-11ea-8686-27bd02222c53.jpg)

Reply:
since it is one item it should still have a title element
`<title>Prayers</title>`
for 1.2.2 - BAVet108 sends to LIT4711Mastabqwe, but it sounds rather like LIT6052MMIntrPrayer to me, here labeled as "መስተብቍዕ" which may therefore remain in the title field

Reply:
(alternatively we could also ungroup them)